src/o365/spo/commands/mail/mail-send.ts

```diff
@@ -0,0 +1,214 @@
+  }
+
+  public get description(): string {
+    return 'Send an email from SharePoint';
+  }
+
+  public getTelemetryProperties(args: CommandArgs): any {
+    const telemetryProps: any = super.getTelemetryProperties(args);
+    telemetryProps.to = typeof args.options.to !== 'undefined';
+    telemetryProps.subject = typeof args.options.subject !== 'undefined';
+    telemetryProps.body = typeof args.options.body !== 'undefined';
+    telemetryProps.from = typeof args.options.from !== 'undefined';
+    telemetryProps.cc = typeof args.options.cc !== 'undefined';
+    telemetryProps.bcc = typeof args.options.bcc !== 'undefined';
+    telemetryProps.additionalHeaders = typeof args.options.additionalHeaders !== 'undefined';
+    return telemetryProps;
+  }
+
+  public commandAction(cmd: CommandInstance, args: CommandArgs, cb: () => void): void {
+    if (this.debug) {
+      cmd.log(`Retrieving access token for the site collection ${auth.service.resource}...`);
+    }
+
+    auth
+      .ensureAccessToken(auth.service.resource, cmd, this.debug)
+      .then((accessToken: string): request.RequestPromise => {
+        if (this.debug) {
+          cmd.log(`Retrieved access token ${accessToken}. Retrieving request digest...`);
+        }
+
+        return this.getRequestDigest(cmd, this.debug);
+      })
+      .then((res: ContextInfo): request.RequestPromise => {
+        if (this.debug) {
+          cmd.log('Response:');
+          cmd.log(res);
+          cmd.log('');
+        }
+
+        const params: any = {};
+        params.properties = {};
+        params.properties.__metadata = { "type": "SP.Utilities.EmailProperties" };
+        params.properties.Body = args.options.body;
+        params.properties.Subject = args.options.subject
+
+        params.properties.To = { results: args.options.to.replace(/\s+/g, '').split(',') };
+        if (this.verbose) {
+          cmd.log(`List of recipients: ${args.options.to}...`);
+        }
+
+        if (args.options.from && args.options.from.length > 0) {
+          params.properties.From = args.options.from;
+          if (this.verbose) {
+            cmd.log(`Mail will send from: ${args.options.from}...`);
+          }
+        }
+
+        if (args.options.cc && args.options.cc.length > 0) {
+          params.properties.CC = { results: args.options.cc.replace(/\s+/g, '').split(',') };
+          if (this.verbose) {
+            cmd.log(`List of addresses to which a carbon copy: ${args.options.cc}...`);
+          }
+        }
+
+        if (args.options.bcc && args.options.bcc.length > 0) {
+          params.properties.BCC = { results: args.options.bcc.replace(/\s+/g, '').split(',') };
+          if (this.verbose) {
+            cmd.log(`List of addresses that receive a copy of the mail but are not listed as recipients: ${args.options.bcc}...`);
+          }
+        }
+
+        if (args.options.additionalHeaders) {
+          params.properties.AdditionalHeaders = args.options.additionalHeaders;
```

Comment:
You're passing additionalHeaders as a string, but according to the API docs it's supposed to be a collection.

<img width="561" alt="image" src="https://user-images.githubusercontent.com/11164679/54074664-94788f00-4295-11e9-8622-5fcd3c5dcd4b.png">